features/old/liquid/active_docs.feature

```diff
@@ -53,7 +53,25 @@ Feature: ActiveDocs
 
     <h3>ActiveDocs version 2.2.10</h3>
 
-    {% include 'shared/swagger_ui' %}
+    <div id="swagger-ui-container" class="swagger-ui-wrap"></div>
```

Comment:
So we do not have it inside the `'shared/swagger_ui'` anymore?
Do we do this because of liquid template migration?

Reply:
I think so. This way it is easier to upgrade and no need to touch `shared/swagger_ui`.